johnwayodi/store-manager
# #161169326 Add Sale Records Page for Admin

#### What does this PR do?
Adds view where the Admin can be able to see sale records from all the various Store Attendants in the application
#### Description of Task to be completed?
Add Filter feature that will allow the Admin to filter records based on Store Attendants
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, **_store-manager_** by default
3. `git checkout ft-sales-records-admin-161169326`
4. open sale-records-admin.html on browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161169326
#### Screenshot
![admin-sales](https://user-images.githubusercontent.com/10683903/46842013-cf07f980-cdd3-11e8-88d3-333da08d3d8e.png)

#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
